examples/tutorials/curation/plot_2_train_a_model.py

```diff
@@ -0,0 +1,125 @@
+# units; :code:`sorting_2` is uncoupled, so should produce noise. We'll combine the two into one sorting
+# object using :code:`si.aggregate_units`.
+#
+# (When making your own model, you should
+# `load your own recording <https://spikeinterface.readthedocs.io/en/latest/modules/extractors.html>`_
+# and `do a sorting <https://spikeinterface.readthedocs.io/en/latest/modules/sorters.html>`_ on your data.)
+
+recording, sorting_1 = si.generate_ground_truth_recording(num_channels=4, seed=1, num_units=5)
+_, sorting_2 =si.generate_ground_truth_recording(num_channels=4, seed=2, num_units=5)
+
+both_sortings = si.aggregate_units([sorting_1, sorting_2])
+
+##############################################################################
+# The models are based on `quality metrics <https://spikeinterface.readthedocs.io/en/latest/modules/qualitymetrics.html>`_
+# and `template metrics <https://spikeinterface.readthedocs.io/en/latest/modules/postprocessing.html#template-metrics>`_,
+# which are computed using a :code:`sorting_analyzer`. So we'll now create a sorting
+# analyzer and compute the extensions needed to get the metrics.
+
+analyzer = si.create_sorting_analyzer(sorting = both_sortings, recording=recording)
+analyzer.compute(['noise_levels','random_spikes','waveforms','templates','spike_locations','spike_amplitudes','correlograms','principal_components','quality_metrics','template_metrics'])
+
+##############################################################################
+# Let's plot the templates for the first and fifth units. The first (unit id 0) belongs to
+# :code:`sorting_1` so should look like a real unit; the sixth (unit id 5) belongs to :code:`sorting_2`
+# so should look like noise.
+
+sw.plot_unit_templates(analyzer, unit_ids=[0,5])
+
+##############################################################################
+# This is as expected: great! Find out more about plotting using widgets `here <https://spikeinterface.readthedocs.io/en/latest/modules/widgets.html>`_. The labels
+# for our units are then easy to put in a list:
+
+labels = ['good', 'good', 'good', 'good', 'good', 'bad', 'bad', 'bad', 'bad', 'bad']
+
+##############################################################################
+# Step 2: Train our model
+# -----------------------
+#
+# With the labelled data, we can train the model using the :code:`train_model` function.
+# Here, the idea is that the trainer will try several classifiers, imputation strategies and
+# scaling techniques then save the most accurate. To save time, we'll only try one classifier
+# (Random Forest), imputation strategy (median) and scaling technique (standard scaler).
+
+folder = "my_model"
+
+# We will use a list of one analyzer here, we would strongly advise using more than one to
+# improve model performance
+trainer = sc.train_model(
+    mode = "analyzers", # You can supply a labelled csv file instead of an analyzer
+    labels = [labels],
+    analyzers = [analyzer],
```

Comment:
I think it might be worth explaining in detail what happens with multiple analyzers, I had to think about it a bit. I guess in reality, you always want to do this. Maybe a short code example underneath?

'In reality, we will usually train a model using multiple analyzers from an experiment. Recall each analyzer contains a set of quality metrics per unit, and we provide the ground-truth (human curated) label for each unit to use in training. Therefore, we have a set of sorting analyzer and corresponding label for each recording in our dataset.

For example, we might load previously generated analyzers and labels from file like:

```
all_labels = []
all_analyzers = []
for name in [some list of animal names]
    analzyer = si.load...
   labels = si.load...
   append..    


```

Reply:
I guess there would need to be a step to ensure all anazyers have the same QMs computed?

Reply:
Yeah, the `train_model` function checks that the same QMs are computed.
I agree about explaining in more detail, but I think the example of loading a list of analyzers is a bit overkill/distracting.

Reply:
Added a small additional sentence.